package.json

```diff
@@ -58,6 +60,7 @@
     "faucet": "0.0.1",
     "gl": "^2.1.5",
     "glslify": "^5.0.2",
+    "glslify-babel": "^1.0.0",
```

Comment:
FWIW, I notice that these tend to be named `babel-plugin-...` But should be OK as is.